openbb_platform/providers/tmx/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "openbb-tmx"
+version = "1.1.1"
```

Comment:
So this is something we need to hammer out @IgorWounds.  Should this technically be 1.0.0 ?

Reply:
Yes, we agreed to start every new extension on 1.0.0.